src/Controller/CRUDController.php

```diff
@@ -1032,7 +1036,7 @@ protected function getBaseTemplate(): string
     /**
      * @throws \Exception
      */
-    protected function handleModelManagerException(\Exception $exception): void
+    protected function handleModelManagerException(\Throwable $exception): void
```

Comment:
This one might get reverted, wdyt @VincentLanglet ?

Reply:
Indeed. This would means Rector introduce BC-break and we can't fully rely on it :/

Reply:
We can disable specific rules that might introduce BC breaks